.github/workflows/fast_testing.yaml

```
@@ -0,0 +1,105 @@
+name: fast_testing
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  linux:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - 'debug-master'
```

It will be nice to test on some release.
```suggestion
        tarantool:
          - '3.1'
          - 'debug-master'
```